libnest/src/system/installer.rs

```diff
@@ -69,14 +69,48 @@ impl<'a, 'b, 'c, 'd> Installer<'a, 'b, 'c, 'd> {
     }
 
     /// Performs the installation.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # extern crate libnest;
+    /// # use std::path::Path;
+    /// // Let's install the `gcc` package
+    /// use libnest::system::System;
+    /// use libnest::system::installer::Installer;
+    /// use libnest::config::Config;
+    /// use libnest::repository::Repository;
+    /// use libnest::package::Package;
+    ///
+    /// let config = Config::new();
+    /// let repository = Repository::new("stable");
+    /// let system = System::current();
+    /// let data = Path::new("/var/nest");
+    ///
+    /// if let Some(category_cache) = repository.cache(&config).category("sys-devel") {
+    ///    if let Some(Ok(manifest_cache)) = category_cache.manifest("gcc") {
+    ///         let package = Package::from(&repository, manifest_cache.manifest().clone());
+    ///         let mut installer = system.installer(&config, &data, &package);
```

Comment:
The path taken by `system.installer` is the path where the data of the package is located. This data is a compressed tarball (`.tar.gz`), not a directory. A valid value could be `/tmp/gcc.tar.gz`